### PR TITLE
[4.0] Namespace finder

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
Namespace the code

### Testing Instructions
Code review and make sure finder aka smart search is working

### CODE REVIEW

File: administrator/components/com_finder/Field/SearchfilterField.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 use Joomla\CMS\Factory;
 use Joomla\CMS\Form\Field\ListField;
+use Joomla\CMS\HTML\HTMLHelper;use Joomla\CMS\Language\Text;
```
Comment (Quy) [inline administrator/components/com_finder/Field/SearchfilterField.php]:
```suggestion
use Joomla\CMS\HTML\HTMLHelper;
use Joomla\CMS\Language\Text;
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5d5933b13908213609086f235c5d6012a444f7ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24691">issues.joomla.org/tracker/joomla-cms/24691</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**